### PR TITLE
Resolve to ESM by default

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Resolve to published ES module by default, aligning with the direction of our other published libraries.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/browser-wallet/src/popup/shared/Form/common/utils.tsx [hunk 1]
```diff
@@ -121,7 +121,7 @@ export const makeUncontrolled = <E, P extends RequiredUncontrolledFieldProps<E>>
         // Filter away all props required for form registration, leaving the props for the input '<Field />'
         const { name, rules, register, onChange: ownOnChange = noOp, onBlur: ownOnBlur = noOp, ...ownProps } = props;
         const registerProps = register(name, rules);
-        const { errors, touchedFields, isSubmitted } = useFormState<TFieldValues>(name);
+        const { errors, touchedFields, isSubmitted } = useFormState<TFieldValues>({ name });
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/shared/Form/common/utils.tsx:124]: :thinking:

File: packages/browser-wallet/src/popup/shared/Form/Form.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ export const useForm = <TFormValues extends FieldValues = FieldValues>(
     props?: UseFormProps<TFormValues>
 ): UseFormReturn<TFormValues> => useFormLib<TFormValues>({ ...useFormDefaults, ...props });
 
-type FormProps<TFormValues> = ClassName & {
+type FormProps<TFormValues extends FieldValues = FieldValues> = ClassName & {
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/shared/Form/Form.tsx:26]: :thinking:

File: packages/browser-wallet-api-helpers/package.json [hunk 3]
```diff
@@ -4,8 +4,17 @@
     "license": "Apache-2.0",
     "packageManager": "yarn@3.2.0",
     "main": "lib/index.js",
-    "browser": "lib/concordiumHelpers.min.js",
+    "browser": "./lib/concordium.min.js",
     "types": "lib/index.d.ts",
+    "cdn": "./lib/concordium.min.js",
```
Comment (shjortConcordium) [inline packages/browser-wallet-api-helpers/package.json:9]: How is this supposed to work, the built file is still named `concordiumHelpers.min.js`?

Comment (soerenbf) [reply packages/browser-wallet-api-helpers/package.json:9]: You're right.. Don't know where I got this from 🤷

File: packages/browser-wallet/src/popup/shared/Form/common/utils.tsx (hunk 1 above)
Comment (soerenbf) [reply packages/browser-wallet/src/popup/shared/Form/common/utils.tsx:124]: I don't know.. It complained after upgrading typescript, and looking at the function signature and the documentation, I don't know why it was accepted prior to this..

https://react-hook-form.com/docs/useformstate

File: packages/browser-wallet/src/popup/shared/Form/Form.tsx (hunk 2 above)
Comment (soerenbf) [reply packages/browser-wallet/src/popup/shared/Form/Form.tsx:26]: Again, upgrading typescript triggered this.

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**